model/src/order.rs

```diff
@@ -162,6 +162,22 @@ pub struct OrderCreation {
     pub signing_scheme: SigningScheme,
 }
 
+#[derive(Clone, Copy, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub struct OrderCreationPayload {
+    #[serde(flatten)]
+    pub order_creation: OrderCreation,
+    pub from: Option<H160>,
+}
+
+impl From<OrderCreation> for OrderCreationPayload {
```

Comment:
Why is this needed? I don't really see the use case for upcasting an order creation to OrderCreationPayload. If this is for the e2e test we should probably provide the `from` address, as this is the first part where this feature will become useful (remember the time you were trying to debug the e2e test because of some signature glitch).

Reply:
Sorry this was left behind from the change in my last commit before backwards compatibility. Will remove.